pkg/rancher-desktop/backend/mock.ts

```diff
@@ -91,7 +91,7 @@ export default class MockBackend extends events.EventEmitter implements VMExecut
 
   noModalDialogs = true;
 
-  requiresRestartReasons(config: RecursivePartial<BackendSettings>): Promise<RestartReasons> {
+  requiresRestartReasons(config: RecursiveReadonly<BackendSettings>): Promise<RestartReasons> {
```

Comment:
Hmm, is this necessary? I believe the interface still calls for `RecursivePartial`?

Reply:
Correct.